### PR TITLE
Implementing repository

### PR DESCRIPTION
**Pull request by Albertoimpl**

### CODE REVIEW

**Review by fifthposition — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -24,6 +24,10 @@ buildscript {
     }
 }
 
+plugins {
+    id "io.franzbecker.gradle-lombok" version '3.2.0'
```
Comment (royclarkson) [inline build.gradle:28]: we might want to be careful about using Lombok in samples. I'm on the fence. I know it makes things easier.

**Review by royclarkson — APPROVED** [review]
Nice work!

File: build.gradle (hunk 1 above)
Comment (Albertoimpl) [reply build.gradle:28]: I went for the easier thing. 
I want to revisit this module once spring-boot-r2dbc gets officially released, happy to tackle this then.

**Comment by Albertoimpl** [comment]
Thanks a lot for reviewing it!

**Pull request merged**